docs/iris/src/_templates/index.html

```diff
@@ -29,7 +29,7 @@
     // Only use first 5 images
     for (var i = 0; i < 5; i++) {
         var img = images[i];
-		$ss.append('<a href="'+img[2]+'"><img src="'+img[1]+'" class="slideshowImage"/></a>');
+		$ss.append('<a href="'+img[2]+'" style="background-color: white;" ><img src="'+img[1]+'" class="slideshowImage"/></a>');
```

Comment:
While I understand what this is doing I don't understand why it's needed – I haven't seen a problem with the slideshow that should require such a CSS change...

Reply:
![bad_background](https://user-images.githubusercontent.com/810663/32223402-168c35dc-be35-11e7-90e8-d734e57af44b.png)


I'm on Chrome for OSX 61.0.3163.100.

Visiting https://scitools-docs.github.io/iris/master/index.html.

Reply:
Ah – note the difference to http://scitools.org.uk/iris/docs/latest/index.html (where the above black bars never appear, so I'll not add an image!).